framework/components/ecl-banners/ecl-banners.config.js

```diff
@@ -1,6 +1,46 @@
 module.exports = {
   title: 'Banners',
   label: 'Banners',
-  status: 'planned',
+  status: 'ready',
   tags: ['molecule'],
+  collated: true,
+  collator(markup, item) {
+    return `
+      <!-- Start: @${item.handle} -->\n
+      <h3 class="ecl-heading ecl-heading--h3">${item.label}\n</h3>\n
+      ${markup}\n
+      <!-- End: @${item.handle} -->\n
+    `;
+  },
+  default: 'background',
+  variants: [
+    {
+      name: 'background',
+      label: 'Background banner',
+      context: {
+        modifier: 'background',
+        src: 'http://lorempixel.com/1600/370/abstract/',
```

Comment:
This will not pass visual regression tests.
Please target always the same image.